src/auto_docking/scripts/auto_docking_client.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+
+import roslib; roslib.load_manifest('kobuki_auto_docking')
+import rospy
+
+import actionlib
+from kobuki_msgs.msg import AutoDockingAction, AutoDockingGoal
+from actionlib_msgs.msg import GoalStatus
+from diagnostic_msgs.msg import DiagnosticArray
+
+
+class AutoDocking:
+    BATTERY_THRESHOLD = 4
```

Comment:
Use [ros parameters](http://wiki.ros.org/rospy/Overview/Parameter%20Server) for these values.